### PR TITLE
Add modified quick-install script for ubuntu 20.04.

### PR DESCRIPTION
**Pull request by ways**
Removed ruby ppa. Updated ruby package names to 2.7. Added version for bundler. Works for me (TM).

### CODE REVIEW

**Comment by willpower232** [comment]
I'm not sure how well Postal supports Ruby 2.7, have you seen #1174?

**Comment by ways** [comment]
Oh, I had not seen that. I got no error messages starting up my postal, but haven't added any accounts yet. Will report back how that goes.

File: script/install/ubuntu2004.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# This will install everything required to run a basic Postal installation.
+# This should be run on a clean Ubuntu 16.04 server.
```
Comment (ilovepancakes95) [inline script/install/ubuntu2004.sh:4]: Update to reflect use on Ubuntu 20.04?

**Comment by ilovepancakes95** [comment]
@ways Any luck with your test?

**Comment by ways** [comment]
Once I added a team, I got errors. So had to reinstall older ruby with renv. =/ Guess this should be closed until a new ruby version is supported.

**Pull request closed without merging**